daemon/cmd/ipam_test.go

```diff
@@ -0,0 +1,28 @@
+package cmd
```

Comment:
Please add the copyright header as found in other files, and make sure to `gofmt` this file. Thanks

Reply:
Done.